test/qa/test_connect.py

```diff
@@ -468,4 +468,73 @@ def test_connect_to_unavailable_servers(tech, proto, obfuscated):
             sh.nordvpn.connect(name)
 
         print(ex.value)
-        assert lib.is_connect_unsuccessful(ex)
\ No newline at end of file
+        assert lib.is_connect_unsuccessful(ex)
+
+
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_status_disconnected(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
+    assert network.is_disconnected()
+    assert "Disconnected" in sh.nordvpn.status()
+
+
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(60)
+def test_status_connected(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
+    with lib.Defer(sh.nordvpn.disconnect):
+        name, hostname = server.get_hostname_by(technology=tech, protocol=proto, obfuscated=obfuscated)
+        sh.nordvpn.connect(hostname.split(".")[0])
+
+        connect_time = int(time.time())
+
+        data_10MB = requests.get("http://speedtest.tele2.net/10MB.zip")
+        destination_file = "file_10MB"
+        if data_10MB.status_code == 200:
+            with open(destination_file, "wb") as file:
+                file.write(data_10MB.content)
+
+        files = {'file': open(destination_file, 'rb')}
+        requests.post("http://speedtest.tele2.net/upload.php", files=files)
+
+        current_time = int(time.time())
```

Comment:
Maybe should be called `status_time, as it's used much further below and is not really "current" anymore then.